Update manage-alert-notification-silencing-rules.md (#23382)
netdata/netdata: docs/alerts-and-notifications/notifications/centralized-cloud-notifications/manage-alert-notification-silencing-rules.md

**Immediate**: From now until turned off or until specific duration (start and end date automatically set).

**Scheduled**: Specify start and end time when the rule becomes active and inactive (time set according to your browser local timezone).

**Recurring**: Set a repeating schedule for the rule to activate and deactivate automatically on a defined cadence. Configure:

- **Starts at**: the date and time of the first occurrence

> - **Lasts until**: the end time of each occurrence (the gap between start and end defines how long each occurrence stays active)

Reviewer: ```suggestion
- **Lasts until**: the end time of the first occurrence (the gap between start and end defines how long each occurrence stays active)
```

- **Repeat**: the recurrence pattern (e.g. weekly on Friday)

> - **Timezone**: your local timezone, so the window follows your wall-clock time even across DST changes

Reviewer: ```suggestion
- **Timezone**: the timezone the schedule is anchored to (typically your local one), so the window follows that zone's wall-clock time even across DST changes
  ```

</details>

<details>
<summary><strong>Recurring Weekly Alert Silencing</strong></summary><br/>

**Use Case**: An alert fires predictably every week during a known window (e.g. a long-running MySQL job every Friday evening to Saturday morning).

**Configuration Steps**:

1. Choose "All users" or "Myself" based on impact
2. Set name: "Weekly MySQL Long-Running Query - Friday Night"
3. **Node Criteria**:
    - Rooms: All Rooms (or the specific room containing the node)

>     - Nodes: [specific node name, e.g. va-stats4dbs-1]

Reviewer: This is a real internal hostname (`va-stats4dbs-1`). The rest of this doc uses generic example names (`child1`), and we shouldn't publish production node names in public docs.

>     - Nodes: [specific node name, e.g. va-stats4dbs-1]

Author: ```suggestion
    - Nodes: [specific node name, e.g. child1]]
```

| Intended stress usage on CPU     | All Rooms          | *             | *                      | *                                              | system.cpu    | *                      | *          | Silences specific Alerts across all nodes and their CPU cores                       |
| Silence role webmaster           | All Rooms          | *             | *                      | *                                              | *             | *                      | webmaster  | Silences all Alerts configured with role webmaster                                  |
| Silence Alert on node            | All Rooms          | child1        | *                      | httpcheck_posthog_netdata_cloud.request_status | *             | *                      | *          | Silences specific Alert on child1 node                                              |
| Disk Space Alerts on mount point | All Rooms          | *             | *                      | disk_space_usage                               | disk.space    | disk_space_opt_baddisk | *          | Silences specific Alert instance on all nodes /opt/baddisk                          |

> | Weekly recurring window          | All Rooms          | va-stats4dbs-1| *                      | [mysql long-running-query alert name]          | *             | *                      | *          | Silences a predictable weekly alert every Friday evening to Saturday morning        |

Reviewer: Here again, we use a real internal hostname, also the alert name should be `mysql_10s_slow_queries`.
```suggestion
| Weekly recurring window          | All Rooms          | db-node-1     | *                      | mysql_10s_slow_queries                         | *             | *                      | *
        | Silences a predictable weekly alert every Friday evening to Saturday morning        |
  ```
